api/src/main/java/org/apache/cloudstack/api/response/NetworkResponse.java

```diff
@@ -556,4 +570,20 @@ public boolean getEgressDefaultPolicy() {
     public void setEgressDefaultPolicy(Boolean egressDefaultPolicy) {
         this.egressDefaultPolicy = egressDefaultPolicy;
     }
+
+    public void setInternetProtocol(String internetProtocol) {
+        this.internetProtocol = internetProtocol;
+    }
```

Comment:
@shwstppr
advice: return internet protocol= "IPv4" in network response by default and display it on UI ?